.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -2,24 +2,37 @@
 name: Bug Report
 about: Create a report to help us improve
 title: ''
-labels: Support
+labels: "Support, needs-triage"
```

Comment:
not sure if we should keep the `Support` label

Reply:
I will remove it.